e2e-tests/cypress/integration/appeal-submission-access-to-appeal-site.feature

```diff
@@ -2,27 +2,36 @@ Feature: Access to the Appeal Site
 I want to notify the Planning Inspectorate if access to the appeal site is restricted so they are aware to contact me for access.
 
     Scenario: Prospective appellant provide access to the appeal site
-        Given the user is prompted to provide access to the inspector visiting the appeal site
-        When the user selects "Yes" to provide access
-        Then the user can see the selected option "Yes is" submitted
-
-     Scenario: Prospective appellant does not provide additional information and access to the appeal site
-        Given the user is prompted to provide access to the inspector visiting the appeal site
-        When the user selects "No" to provide access
-        And the user "does not" provide additional information
-        Then the user is informed that "further information is required to gain access to the restricted site"
-        And the user can see the selected option "is not" submitted
-
-      Scenario: Prospective appellant provide additional information on restricted access to the appeal site
-        Given the user is prompted to provide access to the inspector visiting the appeal site
-        When the user selects "No" to provide access
-        And the user "does" provide additional information
-        Then the user can see the selected option "No is" submitted
+      Given the user is prompted to provide access to the inspector visiting the appeal site
+      When the user selects "Yes" to provide access
+      Then the user can see the selected option "Yes is" submitted
+
+    Scenario: Prospective appellant does not provide additional information and access to the appeal site
+      Given the user is prompted to provide access to the inspector visiting the appeal site
+      When the user selects "No" to provide access
+      And the user "does not" provide additional information
+      Then the user is informed that "further information is required to gain access to the restricted site"
+      And the user can see the selected option "is not" submitted
+
+    Scenario: Prospective appellant provide additional information on restricted access to the appeal site
+      Given the user is prompted to provide access to the inspector visiting the appeal site
+      When the user selects "No" to provide access
+      And the user "does" provide additional information
+      Then the user can see the selected option "No is" submitted
+
+    Scenario: Prospective appellant provide additional information which exceeds the character limit
+      Given the user is prompted to provide access to the inspector visiting the appeal site
+      When the user selects "No" to provide access
+      And the user does provide additional information with character length exceeding the limit
```

Comment:
I don't think we should have an And following a When. The When represents the single trigger event/action.

Reply:
It could read something like...
Given access to appeal site is restricted
When details of the access restriction is provided using more than the allowed character limit
Then the appeal is not updated because the character limit is exceeded

Reply:
It was not what was asked in the site and safety issues template and are we supposed to mix several style in the same feature file ?  Because this phrasing will look really weird compared to the others
<img width="943" alt="Capture d’écran 2021-01-08 à 12 42 22" src="https://user-images.githubusercontent.com/3695996/104011768-f4e8f800-51ae-11eb-825d-e1fa5544862b.png">

Reply:
> It could read something like...
> Given access to appeal site is restricted
> When details of the access restriction is provided using more than the allowed character limit
> Then the appeal is not updated because the character limit is exceeded

Are we supposed to change the style of a current feature file ? Because this wording will look weird compared to the others.

Reply:
Interesting. Strictly speaking it is a ticket to add a scenario so as a minimum the new scenario should be as good as we can do it I think i.e. it should follow our latest best practice. I suppose it boils down to whether 'update feature file...' should include a general refactoring to the latest best practice for the pre-existing scenarios.

Reply:
comments are discussed in chat. Agreed on updating the new scenario to be consistent with the rest of the others